forest_lib/forest/data.py

```diff
@@ -472,21 +419,17 @@ def __init__(self,
 
 
         # set up data loader functions
-        self.loaders = dict([(v1, self.basic_cube_load) for v1 in VAR_NAMES])
-        self.loaders[WIND_SPEED_NAME] = self.wind_speed_loader
-        self.loaders[WIND_STREAM_NAME] = self.wind_speed_loader
+        self.cube_loaders = dict([(v1, self.basic_cube_load) for v1 in VAR_NAMES])
```

Comment:
Could use dictionary comprehension. Small tweak.

`{ var : self.basic_cube_load for var in VAR_NAMES }`